src/app/users/users-achievements/users-achievements.component.ts

```diff
@@ -140,7 +139,9 @@ export class UsersAchievementsComponent implements OnInit {
   }
 
   generatePDF() {
-    const formattedBirthDate = format(new Date(this.user.birthDate), 'MMM d, y');
+    const formattedBirthDate = this.user.birthDate
```

Comment:
Recommend turning this into a `formatDate` function you can put at the top of the file. Cleans up this file and if we want to change the date format later only need to change it in one spot.